code/modules/mob/living/living.dm

```diff
@@ -628,7 +628,6 @@ default behaviour is:
 			if(istype(A,/mob/living/simple_animal/borer) || istype(A,/obj/item/weapon/holder))
 				return
 
-	M.status_flags &= ~PASSEMOTES
```

Comment:
This caused a runtime error when resisting out of a backpack. Passemotes flag is obsolete with my communication fixes now anyways. should look at phasing it out